Lib/fontTools/feaLib/builder.py

```diff
@@ -8,15 +8,19 @@
 import itertools
 
 
-def addOpenTypeFeatures(featurefile_path, font):
-    builder = Builder(featurefile_path, font)
+def addOpenTypeFeatures(font, featurefile_path=None, featurefile_data=None):
```

Comment:
FWIW, I'm ok with `fea_path` and `fea_data` if you prefer having the underscore.